table_filters/omero_filters.py

```diff
@@ -82,8 +104,11 @@ def get_script(request, script_name, conn):
 
         filter_params = [{'name': 'column_name',
                           'type': 'text',
-                          'values': column_names[1:],   # 1st column is Well
-                          'default': column_names[1]},
+                          'values': column_names,
+                          'default': column_names[0],
+                          'minima': minima,
+                          'maxima': maxima,
+                          'histograms': histograms},
```

Comment:
As above, a note here about the special handling of the "column_name" parameter in the UI and that this is tied to the presence of the extra data on *first* parameter might help future understanding.